src/core/zowe/core_for_zowe_sdk/session.py

```diff
@@ -66,9 +65,13 @@ def __init__(self, props: dict) -> None:
         elif props.get("tokenValue") is not None:
             self.session.tokenValue = props.get("tokenValue")
             self.session.type = session_constants.AUTH_TYPE_BEARER
-        elif props.get("cert") is not None and props.get("certKey") is not None:
-            self.session.cert = props.get("cert")
-            self.session.certKey = props.get("certKey")
+        elif props.get("certFile") is not None:
+            if props.get("certKeyFile"):
+                self.session.cert = (props.get("certFile"), props.get("certKeyFile"))
+            else:
+                self.__logger.error("A cert key must be provided")
+                raise Exception("A cert key must be provided")
```

Comment:
Should we enhance this error a bit more to explain why a cert key must be provided?

```suggestion
                self.__logger.error("A certificate key file must be provided when certFile is specified")
                raise Exception("A certificate key file must be provided when certFile is specified")
```